Assets/MixedRealityToolkit/Definitions/InputSystem/MixedRealityInputCapabilities.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+namespace Microsoft.MixedReality.Toolkit.Input
+{
+    /// <summary>
+    /// Platform input capabilities.
+    /// </summary>
+    public enum MixedRealityInputCapabilities
```

Comment:
This is gonna be a pain (sorry) - can you rename this to MixedRealityInputCapability?

The plural here is confusing because it seems to imply that this enum can have more than one, but it's an identifier for a single capability.

Also see MS naming conventions:

https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-classes-structs-and-interfaces#naming-enumerations

It's the first bullet point listed there.

Reply:
FWIW, this is also the case in the rest of our code (aside from a few exceptions which I wish we could change but wouldn't be worth breaking people for)

Reply:
> This is gonna be a pain (sorry) 

Not a pain at all. :)